libr/core/cmd_anal.c

```diff
@@ -6228,6 +6256,7 @@ static void __anal_esil_function(RCore *core, ut64 addr) {
 		eprintf ("Cannot find function at 0x%08" PFMT64x "\n", addr);
 	}
 	r_anal_esil_free (core->anal->esil);
+	core->anal->esil = NULL;
```

Comment:
I think it is highly questionable, that aef uses core->anal->esil as it is also the esil instance that the user uses for emulation-debugging code afaik

Reply:
Totally agree. that we should have multiple instances of esil and kill the instance inside RAnal, otherwise we cant paralelize it